### ISSUE TITLE
Add UI control for the learning rate on the finetuning page.

### ISSUE THREAD

**Issue by Artem-B** [erew123/alltalk_tts#172]
**Is your feature request related to a problem? Please describe.**

The default learning rate does not always work for everyone. E.g. on my sample set, with the default `lr=5e-6`, the loss goes down for the first 4-5 epochs, then starts climbing up, and never goes down after that. Reducing learning rate to `1-e6` results in a continuous decrease in the loss for at least 50 epochs (I didn't train longer than that). Being able to control learning rate is quite useful for finetuning, as it will likely be done on limited inputs with questionable quality, and will need manual tweaking.

**Describe the solution you'd like**
Expose learning rate as a parameter on the finetuning page.

**Describe alternatives you've considered**
The only other option is to manually edit finetune.py and edit the value. It's doable if one knows what they need to do, but it's not a good option for the users who don't know what's under the hood and just want to throw few .wav files on the page, and get the tuned model out.

**Comment by erew123** [comment]
Hi @Artem-B 

This should be in there now.

Thanks
